src/sagemaker/model_monitor/model_monitoring.py

```diff
@@ -1761,7 +1761,7 @@ def _get_default_image_uri(region):
 class BaseliningJob(ProcessingJob):
     """Provides functionality to retrieve baseline-specific files output from baselining job."""
 
-    def __init__(self, sagemaker_session, job_name, inputs, outputs):
+    def __init__(self, sagemaker_session, job_name, inputs, outputs, output_kms_key):
```

Comment:
do we need to default this to `None`? (are we worried about this being a breaking change?)

Reply:
Good call-- defaulted this to `None`.